libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -1176,7 +1176,7 @@ - (void)testInvalidAccessTokenWithInvalidRequest {
 // - sets an invalid refreshToken
 // - issue a valid REST request
 // - ensure all requests are failed with the proper error
-- (void)FIXMEtestInvalidAccessAndRefreshToken {
+- (void)testInvalidAccessAndRefreshToken {
```

Comment:
We had tests for this, but it looks like we had commented them out due to flappy environment issues and forgotten to re-enable them. They were failing before my changes and are passing now.